erica_app/erica/request_processing/requests_controller.py

```diff
@@ -26,7 +27,6 @@ class EricaRequestController(object):
     performing the needed procedures and generating the response. Any request should inherit from this function.
     """
 
-    standard_date_format = "%d.%m.%Y"
```

Comment:
I *think* having this as a constant in e.g. the elsteR_xml_generator could be beneifical.

Reply:
And then using this constant in the ericMapper? Wouldn't it then make more sense to put it as a constant into the ericMapper?

Reply:
Sure. I just thought we might need the date somewhere else. But that's not the case now that I know that UnlockCodeRequest uses as different format. :D

Reply:
I added a constant anyway. I think that makes it a little bit more clear that this is a format that is used in more than one place.